src/Compilers/CSharp/Test/Syntax/Parsing/MemberDeclarationParsingTests.cs

```diff
@@ -4118,8 +4118,7 @@ public void OperatorDeclaration_ExplicitImplementation_21()
                 UsingDeclaration("public int I..operator +(int x) => x;", options: options.WithLanguageVersion(LanguageVersion.Preview),
                     // (1,14): error CS1001: Identifier expected
                     // public int I..operator +(int x) => x;
-                    Diagnostic(ErrorCode.ERR_IdentifierExpected, ".operato").WithLocation(1, 14)
-                    );
+                    Diagnostic(ErrorCode.ERR_IdentifierExpected, ".").WithLocation(1, 14));
```

Comment:
much nicher.  the last errors were just wonky in terms of what they were highlighting!